src/Cluster.jl

```diff
@@ -104,6 +104,13 @@ function init_centroids(X, K, mode)
         permutation = randperm(row)
         idx = permutation[1:K]
         centroids = X[idx, :]
+    
+    elseif mode =="dc"
```

Comment:
Instead of doing something like this wouldn't it be more julian to have multiple dispatch for this function and then depending on the mode type it is a different function.